.github/workflows/deploy-azure-webapps.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     permissions:
       # Needed for Azure login
       id-token: write
-    environment: production
+    environment: development
```

Comment:
the OIDC auth is tied to this Github environment, likely needs update, will try find a link